### ISSUE TITLE
Sidekob::Job renamed to Sidekiq::JobRecord

### ISSUE THREAD

**Issue by dwightwatson** [veeqo/activejob-uniqueness#29]
It appears as the latest Sidekiq release - 6.2.2 - [renames an internal class `Sidekiq::Job` to `Sidekiq::JobRecord`](https://github.com/mperham/sidekiq/blob/master/Changes.md#622).

This change breaks the [Sidekiq patch here](https://github.com/veeqo/activejob-uniqueness/blob/main/lib/active_job/uniqueness/sidekiq_patch.rb#L79).

I'm not entirely sure how to fix this while still supporting previous releases. Below would be a very rough guess and I'm happy to PR if it's suitable.

```rb
if Gem::Version.new(Sidekiq::VERSION) >= Gem::Version.new('6.2.2')
  Sidekiq::Job.prepend ActiveJob::Uniqueness::SidekiqPatch::Job
else
  Sidekiq::JobRecord.prepend ActiveJob::Uniqueness::SidekiqPatch::Job
end
```

**Comment by sharshenov** [comment]
@dwightwatson Thank you for the report. :+1: 
I adapted the fix a bit and added Sidekiq `6.2.2` to the test matrix. https://github.com/veeqo/activejob-uniqueness/pull/30/files
Release `v0.2.1` is coming.

**Comment by dwightwatson** [comment]
Awesome, thanks for the quick turnaround!
